example/src/app/a-bit-more-complex-component/a-bit-more-complex.component.spec.ts

```diff
@@ -0,0 +1,34 @@
+/* tslint:disable:no-unused-variable */
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ABitMoreComplexComponent } from './a-bit-more-complex.component';
+import { ChildComponent } from './child.component';
+
+describe('ABitMoreComplexComponent', () => {
+  let component: ABitMoreComplexComponent;
+  let fixture: ComponentFixture<ABitMoreComplexComponent>;
+
+  beforeEach(
+    async(() => {
+      TestBed.configureTestingModule({
+        declarations: [ABitMoreComplexComponent, ChildComponent],
+      })
+        .overrideComponent(ChildComponent, {set: {template: 'stubbedBody'}})
+        .compileComponents();
+    }),
+  );
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ABitMoreComplexComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('snapshot test', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+});
```

Comment:
The component, test file and the test suite should not just be named "a bit more complex component", we have also other tests than the serializers, so it should definitely explain, that it is a serializer-test.

You can name it something like "complex templating component" or name what is complex about it.

Reply:
Actually, this code was wrote very long time ago, but still... It was "copy-pasted" from other specs in example project.

`example/src/app/calc/calc.component.spec.ts`:

```
  it('should snap', () => {
    expect(fixture).toMatchSnapshot();
  });
```

`example/src/app/simple/simple.component.spec.ts`:
```
  it('snapshots', () => {
    expect(fixture).toMatchSnapshot();
  })
```

And stuff...

I understand your point. Is it critical? If yes, other code have to be refactored too.

Reply:
I see. Still I think "SimpleComponent" alone is clear what to expect in the component. "ABitMoreComplexComponent" will always only stand in comparison to "SimpleComponent", which is not as good.

I would recommend e. g. "MediumComplexComponent"